### PR TITLE
Fix index.md format

### PR DESCRIPTION
**Pull request by blancogerardo94**
Fixed broken code snippets

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/spanish/css/borders/index.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ localeTitle: Propiedad de la frontera
 
 Nuestro atributo personal favorito de CSS, le permite personalizar completamente los bordes que aparecen alrededor de los elementos HTML. Con HTML, solía ser imposible colocar un borde alrededor de un elemento, excepto la tabla. Los bordes de CSS le permiten crear estilos de bordes personalizados y nítidos con muy poco trabajo, en comparación con los métodos anticuados de HTML.
 
-La propiedad abreviada de `border` establece todas las propiedades de borde en una declaración. \`\` \`css borde: 1px sólido # 000;
+La propiedad abreviada de `border` establece todas las propiedades de borde en una declaración. 
+
+```css 
+border: 1px sólido # 000;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]:
```suggestion
border: 1px solid # 000;
```

File: guide/spanish/css/borders/index.md [hunk 2]
```diff
@@ -17,9 +20,8 @@ The properties that can be set, are (in order):
  4. `border-radius` 
  
  It does not matter if one of the values above are missing, for example: 
-```
-
-css borde: rojo sólido;
+```css 
+border: rojo sólido;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]:
```suggestion
border: red solid;
```

File: guide/spanish/css/borders/index.md [hunk 3]
```diff
@@ -41,14 +43,12 @@ The above code is also valid CSS.
  
  Based on the property you choose, these styles can be mismatched. 
  You can style each side seperately: 
-```
-
-css estilo de borde superior: sólido; estilo de borde izquierdo: punteado; estilo de borde derecho: punteado; estilo de borde inferior: doble;
+```css 
+estilo de borde superior: sólido; estilo de borde izquierdo: punteado; estilo de borde derecho: punteado; estilo de borde inferior: doble;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]: this shouldn't be translated at all.

File: guide/spanish/css/borders/index.md [hunk 4]
```diff
@@ -41,14 +43,12 @@ The above code is also valid CSS.
...
 ```
 Or you can style them all at once: 
-```
-
-css estilo de borde: punteado sólido con puntos dobles;
+```css 
+estilo de borde: punteado sólido con puntos dobles;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]: this shouldn't be translated at all.
```suggestion
border: dotted solid double;
```

File: guide/spanish/css/borders/index.md [hunk 5]
```diff
@@ -71,36 +68,38 @@ table { border-width: 7px; border-style: outset; } td { border-width: medium; bo
...
+}
 ```
 ### Border-Radius 
  The `border-radius` property allows the corners of a border to be rounded. This is done by providing a size for 
  how much the border is to be rounded. Size can be in px or %. 
-```
-
-css radio del borde: 25px;
+```css 
+radio del borde: 25px;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]:
```suggestion
border-radius: 25px;
```

File: guide/spanish/css/borders/index.md [hunk 6]
```diff
@@ -71,36 +68,38 @@ table { border-width: 7px; border-style: outset; } td { border-width: medium; bo
...
 ```
 Each corner of `border-radius` can be adjusted. The order is top, bottom, left, right. 
-```
-
-css radio del borde: 15% 10px 30% 5px;
+```css 
+radio del borde: 15% 10px 30% 5px;
```
Comment (lynxlynxlynx) [inline guide/spanish/css/borders/index.md]:
```suggestion
border-radius: 15% 10px 30% 5px;
```

**Review by lynxlynxlynx — DISMISSED** [review]

**Comment by lynxlynxlynx** [comment]
Thanks, it's a useful change.
It would be really great if you went through the rest of the article and untranslated all the remaining css keywords.

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

**Review by RandellDawson — APPROVED** [review]

**Pull request merged**